src/components/orgunits/OrgUnitProfile.js

```diff
@@ -1,9 +1,9 @@
+import { useDataEngine } from '@dhis2/app-runtime'
```

Comment:
Possible to use useDataQuery as this is a react context (with a dynamic query defined above)?